src/engraving/libmscore/beam.cpp

```diff
@@ -1138,12 +1138,12 @@ void Beam::offsetBeamWithAnchorShortening(std::vector<ChordRest*> chordRests, in
     int newDictator = dictator;
     int newPointer = pointer;
     int reduce = 0;
-    while (!isValidBeamPosition(newDictator, isStartDictator, isAscending, isFlat, staffLines, beamCount)) {
+    while (!isValidBeamPosition(newDictator, isStartDictator, isAscending, isFlat, staffLines, false)) {
```

Comment:
I would suggest using a global static constant (you can declare it at the top of the file, and make sure that it will be declared as constexpr) instead of just "false" here in other places

Reply:
Did you one better and just removed it from that function entirely, since the main issue I was trying to solve only needed that value for `isBeamInsideStaff`